# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../hotplug2/destdir.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./docs/Makefile.orig	2007-06-10 12:42:15.000000000 -0400
+++ ./docs/Makefile	2007-06-10 12:43:57.000000000 -0400
@@ -2,12 +2,13 @@
 
 BINS=
 SUBDIRS=
-
+DESTDIR=
+MANDIR=/usr/share/man
 
 all:
 
 install:
-	$(INSTALL) $(wildcard *.8) /usr/share/man/man8/
+	$(INSTALL) $(wildcard *.8) $(DESTDIR)$(MANDIR)/man8/
 
 
 include ../common.mak
--- ./Makefile.orig	2007-06-10 12:41:27.000000000 -0400
+++ ./Makefile	2007-06-10 12:41:57.000000000 -0400
@@ -2,12 +2,13 @@
 
 BINS=hotplug2 hotplug2-dnode
 SUBDIRS=linux24_compat docs examples
+DESTDIR=
 
 
 all: $(BINS)
 
 install:
-	$(INSTALL_BIN) $(BINS) /sbin/
+	$(INSTALL_BIN) $(BINS) $(DESTDIR)/sbin/
 
 
 hotplug2: hotplug2.o childlist.o mem_utils.o rules.o
--- ./linux24_compat/Makefile.orig	2007-06-10 12:45:14.000000000 -0400
+++ ./linux24_compat/Makefile	2007-06-10 12:45:38.000000000 -0400
@@ -2,13 +2,14 @@
 
 BINS=generate_alias hotplug2-coldplug-2.4 hotplug2-modwrap
 SUBDIRS=
+DESTDIR=
 
 
 all: $(BINS)
 
 install:
-	$(INSTALL_BIN) hotplug2-coldplug-2.4 hotplug2-modwrap /sbin/
-	$(INSTALL_BIN) generate_alias /usr/sbin/
+	$(INSTALL_BIN) hotplug2-coldplug-2.4 hotplug2-modwrap $(DESTDIR)/sbin/
+	$(INSTALL_BIN) generate_alias $(DESTDIR)/usr/sbin/
 
 
 hotplug2-coldplug-2.4: hotplug2-coldplug-2.4.o ../parser_utils.o ../filemap_utils.o ../mem_utils.o
--- ./examples/Makefile.orig	2007-06-10 12:44:09.000000000 -0400
+++ ./examples/Makefile	2007-06-10 12:47:48.000000000 -0400
@@ -2,17 +2,19 @@
 
 BINS=
 SUBDIRS=
+DESTDIR=
+KERNELVER=`uname -r`
 
 
 all:
 
 install:
-	case "`uname -r`" in \
+	case "$(KERNELVER)" in \
 		2.6.*) \
-		$(INSTALL) hotplug2.rules-2.6kernel /etc/hotplug2.rules \
+		$(INSTALL) hotplug2.rules-2.6kernel $(DESTDIR)/etc/hotplug2.rules \
 		;; \
 		*) \
-		$(INSTALL) hotplug2.rules-2.4kernel /etc/hotplug2.rules \
+		$(INSTALL) hotplug2.rules-2.4kernel $(DESTDIR)/etc/hotplug2.rules \
 		;; \
 	esac
 
